packages/element/src/platform.js

```diff
@@ -0,0 +1,9 @@
+const Platform = {
+	OS: 'web',
+	select: ( obj ) => ( 'web' in obj ? obj.web : obj.default ),
+};
+/**
+ * Component used to detect the current Platform being used. Use Platform.OS === 'web' to detect if running on web enviroment.
+ * This is the same concept as the React-Native implementation found here: https://facebook.github.io/react-native/docs/platform-specific-code#platform-module
+ */
+export default Platform;
```

Comment:
It would be great to include an example in the docs as well, since this implementation differs a bit, right? Regardless, it can change upstream, so we should document it.

Can we add simple unit tests that would also act like documentation? I personally, often look at unit tests when trying to learn some 3rd party software.

Reply:
@gziolo just added both, give it a look to see if it's ok now :)

Reply:
BTW, I sent a PR upstream

https://github.com/facebook/react-native/pull/26966

I was going to suggest we could get the unit tests from there, but you added them faster 😅

Reply:
BTW, I sent a PR upstream

https://github.com/facebook/react-native/pull/26966

I was going to suggest we could get the unit tests from there, but you added them faster 😅